src/libvirtApi/common.js

```diff
@@ -206,6 +210,21 @@ export function getLibvirtVersion({ connectionName }) {
             .then(version => store.dispatch(updateLibvirtVersion({ libvirtVersion: version[0].v })));
 }
 
+function getCapabilities({ connectionName }) {
+    // Some nodes don"t return all memory in just one cell.
+    // Using -1 == VIR_NODE_MEMORY_STATS_ALL_CELLS will return memory across all cells
+    return call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "GetCapabilities", [], { timeout, type: "" })
+            .then(capabilitiesXML => {
+                const capabilities = parseDumpxmlForCapabilities(capabilitiesXML[0]);
+
+                store.dispatch(setCapabilities({ capabilities }));
+            })
+            .catch(ex => {
+                console.warn("NodeGetMemoryStats failed:", ex.toString());
+                return Promise.reject(ex);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20231030-133824-a492f2d5-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)